index.html

```diff
@@ -19,26 +19,43 @@
 	<script src="https://js.arcgis.com/4.15/"></script>
 
 	<script>
-		var point_template_tree = {
-			title: "{Name}",
-			content: "Cây này ở <b>{Location}</b>."
-		};
-		var point_template_light = {
-			title: "{Name}",
-			content: "Bóng đèn này ở <b>{Location}</b>."
-		};
-		var point_template_chair = {
+		// var point_template_tree = {
+		// 	title: "{Name}",
+		// 	content: "Cây này ở <b>{Location}</b>."
+		// };
+		// var point_template_light = {
+		// 	title: "{Name}",
+		// 	content: "Bóng đèn này ở <b>{Location}</b>."
+		// };
+		// var point_template_chair = {
+		// 	title: "{Name}",
+		// 	content: "Cái ghế này ở <b>{Location}</b>."
+		// };
+		// var point_template_area = {
+		// 	title: "{Name}",
+		// 	content: "Khu vực này ở <b>{Location}</b>."
+		// };
+		// var point_template_university = {
+		// 	title: "{Name}",
+		// 	content: "Trường ở <b>{Location}</b>."
+		// };

```

Comment:
You can remove the commented code, or you can ignore it. So please remove the comment or remove the code.

Reply:
I removed